github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/GitHubPRJobDslExtension.java

```diff
@@ -44,8 +45,20 @@ public Object commitStatusOnGH(Runnable closure) {
                 context.unstableAs(),
                 null,
                 null,
-                null
-        );
+                null);
```

Comment:
that probably wasn't intentional and something eclipse did on it's own to fix whitespace.